hack/generator/pkg/codegen/pipeline_ensure_type_has_arm_type.go

```diff
@@ -45,7 +45,13 @@ func validateAllTypesHaveARMType(definitions astmodel.Types) error {
 
 	var errs []error
 
-	for _, def := range definitions {
+	for name, def := range definitions {
```

Comment:
minor: Wonder if this method name should be changed to `valdiateExpectedTypesHaveARMType` or something?